network/p2p/gossip/bloom.go

```diff
@@ -6,18 +6,45 @@ package gossip
 import (
 	"crypto/rand"
 
+	"github.com/prometheus/client_golang/prometheus"
+
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils"
 	"github.com/ava-labs/avalanchego/utils/bloom"
 	"github.com/ava-labs/avalanchego/utils/math"
 )
 
+type bloomFilterMetrics struct {
+	resetCount prometheus.Counter
```

Comment:
Since we're using similar bloom filter logic for our `network` package as well which uses `utils/bloom` package, maybe we should move the metrics code there?

I think we probably need to consolidate these metrics with the metrics in the [networking](https://github.com/ava-labs/avalanchego/blob/003ed7fbc9b1e959990c1f4f0e2dbdcc32e0e8f3/network/ip_tracker.go#L104-L110) implementation.

Reply:
This is not a bad idea, I gave it a try but I'm not sure utils/bloom should handle metrics. It's totally doable though. Alternatively I can refactor p2p/bloom so that it can be used in `ip_tracker`. We should probably move `ip_tracker` to `p2p/gossip` package in that case. 
cc @StephenButtolph